class.json-api-endpoints.php

```diff
@@ -1246,7 +1247,7 @@ function get_media_item_v1_1( $media_id, $media_item = null, $file = null ) {
 				 * @param array $metadata['sizes'] Array of thumbnail sizes available for a given attachment ID.
 				 * @param string $media_id Attachment ID.
```

Comment:
I don't think so, it shouldn't matter where I pull the $media_id from, it is still the $media_id when the fliter is run.